docs/google/devices.md

```diff
@@ -24,8 +24,179 @@ Several factors can influence this limit, but this can be estimated at about 40
+these gates will silently disappear from the circuit.
+
+#### Physical Z gates
+
+While most applications prefer shorter circuit durations and
+virtual Z gates, Google hardware does offer the possibility of
+applying a physical Z gate that performs a hardware operation
+to affect the frequency of the qubit.
+
+This can be done by applying a PhysicalZTag to the Z gate,
+such as in the following example:
+
+```
+cirq.Z(cirq.GridQubit(5, 5)).with_tags(cirq.google.PhysicalZTag())
+```
+
+### Two Qubit Gates
+
+Google devices provide several options for two-qubit gates.
+The availability of these gates is controlled by the gateset
+parameter that is used.
+
+Note that current hardware gates are noisy and not homogenous
+across the device.  Unitaries provided below are for the ideal case.
+Real unitaries applied to the qubit may vary from qubit to qubit
+and may drift over time.
+
+#### Sycamore Gate
+
+The hardware provides a gate known as the Sycamore gate that can
+be accessed using `cirq.google.SYC`.  This gate is equivalent to
+an FSimGate(π/2, π/6).  That is, it does both a partial swap and
+controlled phase rotation of the |11⟩ state.
+
+The unitary of this gate, which can also be found via the `cirq.unitary`
+function, is:
+
+$$
+\left[
+\begin{matrix}
+1 & 0 & 0 & 0 \\
+0 & 0 & -i & 0 \\
+0 & -i & 0 & 0 \\
+0 & 0 & 0 & e^{-i \pi \over 6}
+\end{matrix}
+\right]
+$$
+
+This gate can be used in `cirq.google.SYC_GATESET` or in the
+`cirq.google.FSIM_GATESET`.
+
+#### Square root of iSWAP
+
+The hardware provides the square root of the iSWAP gate
+using `cirq.ISWAP ** 0.5`.  This gate is equivalent to an FSimGate(-π/4, 0).
+The inverse (`cirq.ISWAP ** -0.5`) is also available
+
+The unitary of this gate, which can also be found via the `cirq.unitary`
+function, is:
+
+$$
+\left[
+\begin{matrix}
+1 & 0 & 0 & 0 \\
+0 & \frac{1}{\sqrt{2}} & \frac{i}{\sqrt{2}} & 0 \\
+0 & \frac{i}{\sqrt{2}}& \frac{1}{\sqrt{2}} & 0 \\
+0 & 0 & 0 & 1
+\end{matrix}
+\right]
+$$
+
+This gate can be used in `cirq.google.SQRT_ISWAP_GATESET` or in the
+`cirq.google.FSIM_GATESET`.
+
+#### CZ gate
+
+The controlled-Z gate `cirq.CZ` is experimentally available on some
+devices.  Be sure to check with your sponsor or in the device specification
+to see if it is available on the processor you are using.
+
+This gate is equivalent to FSimGate(0, π)
+
+#### FSim gateset
+
+The `cirq.FSIM_GATESET` provides all three of the above gates in one set.
+In addition, by using this combined gate set, the FSimGate can be parameterized,
+which allows for efficient sweeps across varying two-qubit gates.
+Note that providing a theta/phi combination that
+is not one of the above gates will cause an error when run on hardware.
+
+
+### Wait gate
+
+For decay experiments and other applications, a WaitGate is provided
+that causes the device to idle for a specified amount of time.
+This can be accomplished by specifying a `cirq.WaitGate`.
```

Comment:
What is the behavior of wait gate when run with other gates? Will the moment last the maximum of the gates in that moment, including a wait gate?

Reply:
Yes, I believe so.  I clarified that for the general case in the moment section above (that the duration of a moment is its longest gate) but did not specifically call it out in this section.